src/main/kotlin/main.kt

```diff
@@ -0,0 +1,20 @@
+import domain.CoordinateRandomGenerator
+import domain.LandMineRandomMap
+import dto.CoordinateGeneratorRequest
+import dto.LandMineMapRequest
+import view.InputView
+import view.LandMineMapUI
+
+fun main() {
+    val height = InputView.height()
+    val width = InputView.width()
+    val landmine = InputView.landmine()
+
+    val landMineMapRequest = LandMineMapRequest.of(height, width)
+    val coordinateGeneratorRequest = CoordinateGeneratorRequest.of(landMineMapRequest.width, landMineMapRequest.height, landmine)
```

Comment:
`CoordinateGeneratorRequest`를 생성할때 전달받는 `width`, `height` 또한 0이상임을 보장하려면 `landMineMapRequest`를 직접 입력받는게 더 방어적일수 있을것 같습니다.


```suggestion
    val landMineMapRequest = LandMineMapRequest.of(height, width)
    val coordinateGeneratorRequest = CoordinateGeneratorRequest.of(landMineMapRequest, landmine)
```